test/infra-stack-props.test.ts

```diff
@@ -44,8 +44,7 @@ test('Throw error on incorrect JSON for opensearch', () => {
   } catch (error) {
     expect(error).toBeInstanceOf(Error);
     // @ts-ignore
-    expect(error.message).toEqual('Encountered following error while parsing additionalConfig json parameter: '
-      + 'SyntaxError: Unexpected token e in JSON at position 33');
+    expect(error.message).toContain('Encountered following error while parsing additionalConfig json parameter:');
```

Comment:
Why did we remove the SyntaxError?

Reply:
the exact position changed on my new mac which means it is not guaranteed. Changed it to `contains`